import-export-cli/integration/devFirst_test.go

```diff
@@ -357,3 +357,55 @@ func TestUpdateDocAndImageOfAPIOfExistingAPI(t *testing.T) {
 	//Validate that document has been updated
 	testutils.ValidateAPIWithIconIsExported(t, args, testutils.DevFirstDefaultAPIName, testutils.DevFirstDefaultAPIVersion)
 }
+
+// Test sequence update as a super tenant user with Internal/devops role
+func TestAPISequenceUpdateWithDevopsSuperTenantUser(t *testing.T) {
+	devopsUsername := devops.UserName
+	devopsPassword := devops.Password
+	apim := apimClients[1]
+	projectName := base.GenerateRandomName(16)
+
+	args := &testutils.InitTestArgs{
+		CtlUser:   testutils.Credentials{Username: devopsUsername, Password: devopsPassword},
+		SrcAPIM:   apim,
+		InitFlag:  projectName,
+		OasFlag:   testutils.TestOpenAPI3DefinitionPath,
+		APIName:   testutils.DevFirstDefaultAPIName,
+		ForceFlag: true,
+	}
+
+	// Initialize the project
+	testutils.ValidateInitializeProjectWithOASFlag(t, args)
+
+	// Add custom sequence file to created project
+	projectPath, _ := filepath.Abs(projectName)
+	srcPathForSequence, _ := filepath.Abs(testutils.DevFirstSampleCaseSequencePath)
+	destPathForSequence := projectPath + testutils.DevFirstSampleCaseDestSequencePathSuffix
+	base.CreateDir(projectPath + testutils.CustomSequenceDirectory)
+	base.Copy(srcPathForSequence, destPathForSequence)
+
+	// Update api.yaml file of initialized project with sequence related metadata
+	apiMetadataYamlPath := projectPath + testutils.DevFirstSampleCaseApiMetadataPathSuffix
+	inSequenceStr := "inSequence: " + testutils.CustomSequenceName
+	base.AppendStringToFile(inSequenceStr, apiMetadataYamlPath)
+
+	// Import the project with custom sequence
```

Comment:
Shall we mention that these sequences are verified ones in the comment so that in the future when we implement a test case for an invalid sequence import use case it will be more clear?

Reply:
+1

Reply:
Fixed with https://github.com/wso2/product-apim-tooling/pull/797/commits/92bad978f4726a199ea81e9b8619bc23239ea4bd